packages/pn-personafisica-webapp/src/pages/NuovaDelega.page.tsx

```diff
@@ -88,7 +88,7 @@ const NuovaDelega = () => {
     void dispatch(createDelegation(values));
     trackEventByType(TrackEventType.DELEGATION_DELEGATE_ADD_ACTION);
   };
-
+  const [inputValue, setInputValue] = useState('');
```

Comment:
Use more specific naming (same as commented in SpecialcCntacts.tsx)

Reply:
do it